### PR TITLE
Improve the GRDB demo.

### PR DESCRIPTION
**Pull request by mbrandonw**
Related to #14, we can leverage `prepareDependencies` to push the database dependency into the `GRDBQueryKey` "library" code.

### CODE REVIEW

File: Examples/GRDBDemo/GRDB/GRDBQueryKey.swift [hunk 1]
```diff
@@ -3,6 +3,39 @@ import GRDB
...
+        @main
+        struct EntryPoint: App {
+          init() {
+            prepareDependencies {
+              $0.defaultDatabase = DatabaseQueue(…)
+            }
+          }
+        
+          // ...
+        }
```
Comment (stephencelis) [inline Examples/GRDBDemo/GRDB/GRDBQueryKey.swift:28]: Let's indent this to set it apart:

```suggestion
            @main
            struct EntryPoint: App {
              init() {
                prepareDependencies {
                  $0.defaultDatabase = DatabaseQueue(…)
                }
              }

              // ...
            }
```

**Comment by acosmicflamingo** [comment]
Good thinking! By the way, since `Examples` is not shipped with the app, I don't think there'd be any harm in making some code in `GRDBQueryKey` public. That way when I copy and paste the code into my own codebase, I don't have to be regularly adding `public`.

**Comment by mbrandonw** [comment]
@acosmicflamingo You can always maintain your own re-usable library of these types and make them public.

**Comment by acosmicflamingo** [comment]
@mbrandonw true! Thanks for the PR!!!

**Pull request merged**